api/scpca_portal/config/local.py

```diff
@@ -17,11 +18,19 @@ class Local(Common):
     AWS_REGION = None
 
     # AWS S3
-    TEST_INPUT_BUCKET_NAME = "scpca-portal-public-test-inputs/2024-09-10/"
-    AWS_S3_INPUT_BUCKET_NAME = TEST_INPUT_BUCKET_NAME if Common.TEST else "scpca-portal-inputs"
+    AWS_S3_INPUT_BUCKET_NAME = "scpca-portal-inputs"
     AWS_S3_OUTPUT_BUCKET_NAME = "scpca-local-data"
 
     CSRF_TRUSTED_ORIGINS = ["localhost", "127.0.0.1"]
 
     # This is only needed locally because everything else will use S3.
     DEV_HOST = os.getenv("DEV_HOST")
+
+    # Code Paths
+    CODE_PATH = Path("/home/user/code")
+
+    DATA_PATH = CODE_PATH / "data"
+    INPUT_DATA_PATH = DATA_PATH / "input"
+    OUTPUT_DATA_PATH = DATA_PATH / "output"
+
+    TEMPLATE_PATH = CODE_PATH / "scpca_portal" / "templates"
```

Comment:
```suggestion
    INPUT_DATA_PATH = Path("/home/user/code/data/input")
    OUTPUT_DATA_PATH = Path("/home/user/code/data/input")
    TEMPLATE_PATH = Path("/home/user/code/scpca_portal/templates")
```

I think we will be better served by removing the path concats and then explicitly defining the location. This change would apply to the other config files as well. I think it made more sense to keep the intermediate variables when we were evaluating these but we don't benefit from that anymore.